code/modules/clothing/suits/armor_vr.dm

```diff
@@ -59,7 +59,7 @@
 
 /obj/item/clothing/suit/armor/combat/USDF
 	name = "marine body armor"
-	desc = "When I joined the Corps, we didn't have any fancy-schmanzy armor. We had sticks! Two sticks, and a rock for the whole platoon�and we had to <i>share</i> the rock!"
+	desc = "When I joined the Corps, we didn't have any fancy-schmanzy armor. We had sticks! Two sticks, and a rock for the whole platoon�and we had to <i>share</i> the rock!"
```

Comment:
VSCode did a fucky wucky

Reply:
That's not vscode, that character was a magical MS word long hyphen before. I'll fix it with a normal hyphen.